util/config/config.go

```diff
@@ -43,6 +43,7 @@ const (
 	DonationAddress              = "NKNaaaaaaaaaaaaaaaaaaaaaaaaaaaeJ6gxa"
 	DonationAdjustDividendFactor = 1
 	DonationAdjustDivisorFactor  = 2
+	GlobalAccountAddress         = "NKNBV71h5oUBjknMHPikPU7nraw9KtksWjtB"
```

Comment:
We should also check and make sure no transfer is made to this address

Reply:
We could, why though? Sending here is effectively burning coins. There is a lot of addresses (public keys) for which nobody will have private key (all zeros, zeros + 1 at the end, etc), we can't possibly check them all.

Reply:
I was trying to make sure that no non-leaf node in the the state trie stores account state (though this is probably not necessary at all)

Reply:
I think we can safely forget about it :)